### ISSUE TITLE
Reference Issue

### ISSUE THREAD

**Issue by adamreed90** [Overmiind/Puppeteer-sharp-extra#19]
Getting this error just trying the quickstart code out. 

`Error CS0012 The type 'Browser' is defined in an assembly that is not referenced. You must add a reference to assembly 'PuppeteerSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.` 

We have PuppeteerSharp v3 installed and working, this error only shows when trying to use this plugin.

**Comment by juergenr** [comment]
Same issue after updating PuppeteerSharp to version 3.0, I  get the Error CS0012 on .
```
            var extra = new PuppeteerExtra();
            var browser = await extra.LaunchAsync(new LaunchOptions { 
                Headless = false,   
                Devtools = true,    
                DefaultViewport = null
            }); 
```
It seems Puppeteer-sharp-extra is no longer compatible with the newest PuppeteerSharp version.

**Comment by Overmiind** [comment]
Hi, I have fixed it in the next version which will be published in a few days

**Comment by juergenr** [comment]
Thanks, for your great work.

**Comment by Overmiind** [comment]
Fixed by #20. Please update your NuGet package to version 1.3.0
